web/client/components/data/query/CrossLayerFilter.jsx

```diff
@@ -63,7 +74,9 @@ export default ({
         title={<Message msgId="queryform.crossLayerFilter.title" />} >
         <Row className="inline-form filter-field-fixed-row">
             <Col xs={6}>
-                <div><Message msgId="queryform.crossLayerFilter.targetLayer"/></div>
+                <div>
+                    { renderUnMatchingLayersInfo() }
+                    <Message msgId="queryform.crossLayerFilter.targetLayer"/></div>
```

Comment:
Move it after

```suggestion
                    <Message msgId="queryform.crossLayerFilter.targetLayer" /> 
                    {renderUnMatchingLayersInfo()}
                </div>
```

Final result looks like this:
![image](https://user-images.githubusercontent.com/1279510/111148966-8fd5c800-858c-11eb-9c88-4f7f90ba2b97.png)